crates/project_panel/src/project_panel.rs

```diff
@@ -2625,6 +2630,14 @@ impl ProjectPanel {
             marked_selections: selections,
         };
 
+        let (bg_color, border_color) = match (is_hovered, is_marked || is_active, self.mouse_down) {
+            (true, _, true) => (item_colors.marked_active, item_colors.hover),
+            (true, false, false) => (item_colors.hover, item_colors.hover),
+            (true, true, false) => (item_colors.hover, item_colors.marked_active),
+            (false, true, _) => (item_colors.marked_active, item_colors.marked_active),
+            _ => (item_colors.default, item_colors.default),
+        };
```

Comment:
The idea is to keep a state for all the events and not do dynamic style adjustments with `.hover(|style| ...`, so we can pass the same color to the background and to the knockout icons. I think this logic replicates exactly the previous behaviour.